common/lib/capa/capa/responsetypes.py

```diff
@@ -833,9 +939,32 @@ def get_score(self, student_answers):
         """
         # log.debug('%s: student_answers=%s, correct_choices=%s' % (
         #   unicode(self), student_answers, self.correct_choices))
+        
+        tree = self.xml
+        problem_xml = tree.xpath('.')
+
+        # Partial credit type - can set 'points' only at the moment.
+        credit_type = problem_xml[0].get('partial_credit', default=False)
+
+        try:
+            credit_type = str(credit_type).lower().strip()
+        except ValueError:
+            _ = self.capa_system.i18n.ugettext
```

Comment:
You're defining it within the `except` clause, which means it's undefined in the rest of the function. What I'm saying is, move the definition to outside of the except clause, then it's defined all the time not just sometimes. That way if new strings are added `_` doesn't have to be defined in a ton of places.